LayoutTests/platform/glib/tables/bugs/bug1163-1-expected.txt

```diff
@@ -6,7 +6,7 @@ layer at (0,0) size 800x600
       RenderBlock (anonymous) at (0,0) size 784x18
         RenderText {#text} at (0,0) size 4x17
           text run at (0,0) width 4: " "
-      RenderTable {TABLE} at (0,18) size 210x92 [border: (3px solid #0000BB)]
```

Comment:
You glib test expectations seem to be in the wrong folder, you dropped `mozilla` from the path for some reason.